src/events/components/Attendance/CaptchaModal.tsx

```diff
@@ -10,10 +10,11 @@ interface ICaptchaModalProps {
   toggleModal: () => void;
   setCaptcha: (token: string | null) => void;
   errorText?: React.ReactNode | string;
+  onError: (error: Error) => void;
```

Comment:
React-struktur spørsmål: hvorfor driver du å sender logikk via to nivåer? Syntes det bare kompliserer kontroll-flyten.

Ser dette ganske ofte, og syntes det fremstår som et veldig anti-pattern?

Reply:
Enig i at flyten er litt komplisert her. Godt mulig at det kan gjøres bedre. Er fort en case av premature optimalization. Sånn det er nå er ikke <CaptchaModal/> knyttet til event-påmelding. Det er en reusable komponent som potensielt kan brukes andre steder. Så da blir det jo sånn at event handlers og tekst som kan være forskjellig fra case til case må være i parent komponenten. 

IMO kunne hele greia vært i AttendButton istedenfor at det er en egen komponent. Eller bare gjort den helt spesifikk for event registration.

Synes forøvrig `setCaptcha` er et dårlig navn her da, som gjør det litt ekstra komplisert. Kunne vært `onValidCaptchaToken` e.l.

Reply:
Synes det ble veldig rotete med feilmeldingene fra parent component og fra Turnstile onError her da. Men siden vi ikke har store ambisjoner for videreutvikling på owf tenker jeg at sånne ting er greit. As long as it works og ikke er helt uforståelig liksom